src/internal_events/amqp.rs

```diff
@@ -5,15 +5,42 @@ pub mod source {
     use vector_core::internal_event::InternalEvent;
 
     #[derive(Debug)]
-    pub struct AMQPEventReceived {
+    pub struct AMQPBytesReceived {
         pub byte_size: usize,
+        pub protocol: &'static str,
     }
 
-    impl InternalEvent for AMQPEventReceived {
+    impl InternalEvent for AMQPBytesReceived {
         fn emit(self) {
-            trace!(message = "Received one event.", internal_log_rate_secs = 10);
-            counter!("processed_events_total", 1);
-            counter!("processed_bytes_total", self.byte_size as u64);
+            trace!(
+                message = "Bytes received.",
+                byte_size = %self.byte_size,
+                protocol = %self.protocol,
+            );
+            counter!(
+                "component_received_bytes_total",
+                self.byte_size as u64,
+                "protocol" => self.protocol,
+            );
+        }
+    }
+
+    #[derive(Debug)]
+    pub struct AMQPEventsReceived {
+        pub byte_size: usize,
+    }
+
+    impl InternalEvent for AMQPEventsReceived {
+        fn emit(self) {
+            trace!(message = "Events received.", internal_log_rate_secs = 10);
```

Comment:
I _think_ trace messages are not supposed to be rate limited ?

Reply:
```suggestion
            trace!(message = "Events received.");
```
It'll be noisy for sure, but yeah - not rate limiting is the guideline there.